curr_scraper/apps/api_caller/call_logic.py

```diff
@@ -0,0 +1,54 @@
+import json
+import os
+from datetime import datetime
+
+import pytz
+import requests
+
+from api_caller import models
+
+
+def call_api():
+    oe_url = os.environ.get('OPEN_EXCHANGE_URL')
+    oe_app_id = os.environ.get('OPEN_EXCHANGE_APP_ID')
+    oe_complete_url = f'{oe_url}/latest.json?app_id={oe_app_id}'
+    try:
+        response = requests.get(oe_complete_url)
+        json_data = json.loads(response.text)
+        base = json_data["base"]
+    except KeyError:
+        print("KeyError")
+    else:
+        return json_data
+
+
+def initialize_currency_table():
```

Comment:
Please add docstrings as per [PEP-257](https://peps.python.org/pep-0257/)

Add docstrings for each function